pkg/status/status.go

```diff
@@ -417,3 +449,33 @@ func (NoOpReporter) StartReporting() <-chan struct{} {
 func (NoOpReporter) ReportMigration() error {
 	return nil
 }
+
+// checkOperatorUpgradeablity checks for the presence of CatalogSourceConfigs,
+// and OperatorSources in the cluster. Since both the APIs are scheduled to be
+// deprecated/are already deprecated, checkOperatorUpgradeablity returns false
+// if it detects the presence of any CSC or Opsrc in the cluster to indicate
+// that the cluster is not upgradable to future versions.
+func checkOperatorUpgradeablity() (bool, error) {
```

Comment:
Shouldn't we be ignoring the default operator sources though? Those will be migrated to catalog sources automatically, so we don't want to block upgrade for those

Reply:
@kevinrizza that is right. Updated to ignore default operator sources.